src/Cardano/CoinSelection/Random.hs

```diff
@@ -106,35 +109,38 @@ data TargetRange = TargetRange
 -- that a randomly chosen UTxO entry will push the total above the upper bound
 -- we set.
 -- @
-random
-    :: forall m e. MonadRandom m
+randomImprove :: MonadRandom m => CoinSelectionAlgorithm m e
+randomImprove = CoinSelectionAlgorithm payForOutputs
```

Comment:
`randomImprove` immediately associates t me with `improve` below. but that is minor

Reply:
My motivation here was to make the names of identifier match up with the names of algorithms. So we have:

1. "Largest-First" ↔ `largestFirst`.
2. "Random-Improve" ↔ `randomImprove`.